### PR TITLE
Add npm run-script build

### PR DESCRIPTION
**Pull request by themalkolm**
* split webspack into two configs
* dev config is still default
* use webpack from local install

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,8 +4,13 @@
...
   "repository": "https://github.com/iverberk/nomad-ui",
   "main": "src/main.js",
+  "bin": {
+    "webpack": ".bin/webpack",
+    "webpack-dev-server": ".bin/webpack-dev-server"
+  },
   "scripts": {
-    "start": "webpack-dev-server"
+    "start": "webpack-dev-server",
+    "build": "webpack --config webpack-prod.config.js -p --progress"
```
Comment (iverberk) [inline package.json]: I'm not too familiar with the npm scripts but will these commands actually reference the commands in the "bin" section above?

Comment (themalkolm) [inline package.json]: Yes.

File: package.json [hunk 2]
```diff
@@ -4,8 +4,13 @@
   "description": "User interface for the HashiCorp Nomad scheduler.",
   "repository": "https://github.com/iverberk/nomad-ui",
   "main": "src/main.js",
+  "bin": {
+    "webpack": ".bin/webpack",
```
Comment (iverberk) [inline package.json]: I only have a node_modules/.bin directory locally. Is that what you want? The npm build command currently doesn't do anything, just exits right away with error code 0.

Comment (themalkolm) [inline package.json]: `npm run-script build`

File: README.md [hunk 3]
```diff
@@ -36,7 +36,7 @@ If you have a Node and Go environment you can also build the production version
 
 ```
 npm install
-NODE_ENV=production webpack -p --progress
+npm run-script build
```
Comment (themalkolm) [inline README.md]: It is mentioned here. Maybe there is a better fix to make npm build work as well.

Comment (themalkolm) [inline README.md]: Anyway it is better than to ask user to have webpack installed in the system as this way we control version and configuration.

File: package.json (hunk 2 above)
Comment (iverberk) [inline package.json]: ah yes, that works, excellent.

File: README.md (hunk 3 above)
Comment (iverberk) [inline README.md]: Definitely!

**Comment by iverberk** [comment]
Thanks!

**Pull request merged**